frontend/src/components/authchooser/index.tsx

```diff
@@ -261,9 +261,14 @@ export function PureAuthChooser({
             <Box alignItems="center" textAlign="center">
               <Box m={2}>
                 <Empty>
-                  {t('Failed to get authentication information: {{ errorMessage }}', {
-                    errorMessage: error!.message,
-                  })}
+                  {error && error.message === 'Bad Gateway'
+                    ? t(
+                        "Please make sure the Kubernetes cluster is running and accessible, Recieved '{{ errorMessage }}' when trying to connect",
```

Comment:
There's a typo, and this can be simplified:

```suggestion
                        "Failed to connect. Please make sure the Kubernetes cluster is running and accessible. Error: {{ errorMessage }}",
```